components/build/pipelines-as-code/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+resources:
+- install.yaml
```

Comment:
What is difference between this and https://github.com/openshift-pipelines/pipelines-as-code/tree/main/config ? Why not to include it directly from git?

Reply:
This is a dump of the release.yaml . Right, I could probably use https://github.com/openshift-pipelines/pipelines-as-code/tree/release-0.5.2/config as well 🤷‍♂️

Reply:
Ah this won't work because https://github.com/openshift-pipelines/pipelines-as-code/tree/main/config  isn't a kustomize directory.

Reply:
yeah it's a ko one

Reply:
In that case use - `https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-0.5.2/release-0.5.2.yaml`

Reply:
@Michkov As a kustomize reference, or a copy-paste? :)

Reply:
Reference to the yaml file, it's clear what is installed and in which version.
```
resources:
- https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/release-0.5.2/release-0.5.2.yaml
```

Reply:
Ah gotcha.
How does this look ? https://github.com/redhat-appstudio/infra-deployments/pull/115/commits/4effd7c8e036b766c534629b5feb4d4ad2d148b3